client/src/pages/TreeDetails/AdoptLikeCheckboxes.js

```diff
@@ -61,23 +63,38 @@ function Adopted({
   );
 }
 
-export default function AdoptLikeCheckboxes({ currentTreeId, common, edit }) {
+export default function AdoptLikeCheckboxes({
+  currentTreeData, edit, isTreeQueryError,
+}) {
   const { user } = useAuth0();
   const mutateTreeLikes = useTreeLikesMutation();
   const mutateTreeAdoptions = useTreeAdoptionsMutation();
   const mutateHistory = useTreeHistoryMutation();
+  const mutateCreateTreeData = useCreateTreeDataMutation();
   const [directions, showDirections] = useState(false);
 
   const handleChange = (event) => {
+    if (isTreeQueryError) {
+      mutateCreateTreeData.mutate({
+        ...currentTreeData,
+        scientific: currentTreeData.scientific
+        || currentTreeData.genus,
+        city: currentTreeData.city || currentTreeData.sourceId,
+        url: currentTreeData.download,
+        volunteer: user.nickname,
+        health: currentTreeData.health || 'fair',
+      });
+    }
```

Comment:
But why overwrite any of those fields, even if you need to create a new tree?  It should just send what's in `currentTreeData`, right?  This would replace the volunteer who planted the tree with the person adopting it, I think.